### PR TITLE
Ggr change intergration

### PR DESCRIPTION
**Pull request by Georges52**
Implementation of the whole Update Standard Change from Endpoint to Backendsystem. Including error handling.

### CODE REVIEW

**Review by vangual — CHANGES_REQUESTED** [review]
Needs minor safety fixes to avoid exception crashes. See my comments and the suggestions attached to them.

File: apiserver/openapi/oim_itsm.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@ paths:
             application/json:
               schema:
                 $ref: "#components/schemas/update_change_response"
+        500:
+          description: "Server error"
+          content:
+            application/json:
+              schema:
+                $ref: "#components/schemas/update_change_response"
```
Comment (vangual) [inline apiserver/openapi/oim_itsm.yaml:27]: This seems wrong, the content of the error contains the same variables as a normal result? Surely not... I'd recommend removing the content key here.

**Review by vangual — CHANGES_REQUESTED** [review]
Forgot to check the yaml content in previous review.

File: apiserver/openapi/oim_itsm.yaml (hunk 1 above)
Comment (Georges52) [reply apiserver/openapi/oim_itsm.yaml]: Is changed

File: apiserver/itsm/calls_change.py [hunk 2]
```diff
@@ -18,16 +18,21 @@ def update_change(body):
...
         "ticketno": myticketno,
         "changeOwnerGroup": mysupportgroup
     }
 
+    MyChange = ValuemationHandler()
     lRet = MyChange.update_change(params)
 
-    retStr = {"ticketno": lRet['data']['ticketno'], "changestatus": lRet['score']}
-    return retStr, 200
+    if lRet['score'] == 'success':
```
[the remark this answers is not in the document]
Comment (Georges52) [reply apiserver/itsm/calls_change.py]: Is changed

File: apiserver/itsm/calls_change.py [hunk 3]
```diff
@@ -18,16 +18,21 @@ def update_change(body):
...
+        retStr = {"ticketno": lRet['data']['ticketno'], "changestatus": lRet['score']}
+        return retStr, 200
+    elif lRet['score'] == 'danger':
```
[the remark this answers is not in the document]
Comment (Georges52) [reply apiserver/itsm/calls_change.py]: Is changed

File: apiserver/workflows/steps/WorkflowSteps.py [hunk 4]
```diff
@@ -132,9 +130,12 @@ def execute(self, context: WorkflowContext):
 
         myChange = ValuemationHandler(myChangeDetails)
         lRet = myChange.create_change()
-        print("lRet:[", lRet, "]")
+        if lRet == "11":
+            logger.error("create of CR is failed:{}".format(lRet))
+        else:
+            crnr = lRet['data']['ticketno']
```
[the remark this answers is not in the document]
Comment (Georges52) [reply apiserver/workflows/steps/WorkflowSteps.py]: Is changed

**Review by vangual — APPROVED** [review]
LGTM

**Pull request merged**